modules/installation-osp-verifying-cluster-status.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-verifying-cluster-status_{context}"]
+= Verifying cluster status
```

Comment:
@maxwelldb, what's the point of this? I don't see what user goal this is supporting.

Reply:
Verify this in coming PR. It was important at some point, at least. :)